build.gradle

```diff
@@ -25,9 +25,14 @@ compileJava {
 minecraft {
     version = "1.12.2-14.23.5.2847"
     runDir = "run"
-    mappings = "stable_39"
+    mappings = "snapshot_20171003"
```

Comment:
This change should not be made

Reply:
I had to or I couldn't get galacricraft to work